third_party/move/move-vm/transactional-tests/tests/function_values_safety/compatibility.masm

```diff
@@ -1,54 +1,87 @@
 // ========= Field Type
 
-//# publish --verbose
-module 0x66::incompatible_field_type
+{% for ty1 in ["u64", "&u8"] %}
```

Comment:
More of a general remark: we probably should try to strike balance between templates, large files and minimal test cases. E.g., for upgradability, it is nice to have an own file for specific rule so we can ensure this path is covered, otherwise the exp file is very verbose it is hard to match inputs to outputs even with lines and commands printed. E.g. here we test field abilities, function types, etc - seems like each can go in its own file in `compatibility` folder

Reply:
Done